modules/300-prometheus/templates/grafana/secret-datasources-list.yaml

```diff
@@ -54,16 +41,6 @@ datasources:
   isDefault: true
   {{- include "authorization_config" (list . (include "main_scrape_interval" . )) | nindent 2 }}
 
-  {{- if ne (int .Values.prometheus.longtermRetentionDays) 0 }}
```

Comment:
We cannot delete this data source. If we do, users will lose the ability to retrieve data from Prometheus LTS. I suggest updating the aggregation proxy and adding a new server group for access to Prometheus LTS, similar to how it was configured in Trickster. Finally, we should update this data source accordingly.